docs/rst/docker/docker_demo/fast_dds_suite.rst

```diff
@@ -0,0 +1,137 @@
+.. _fast_dds_suite:
+
+Fast DDS Suite
+==============
+
+This Docker image contains the complete Fast DDS suite. This includes:
+
+- :ref:`eProsima Fast DDS libraries and examples <fast_dds_suite_examples>`: Fast DDS libraries bundled with several
+  examples that showcase a variety of capabilities of eProsima's Fast DDS implementation.
```

Comment:
I would add here a link to Fast DDS documentation and I would remove the link to the Monitor documentation below. And maybe here a more detailed description of Fast DDS is required whereas the detailed description for the Monitor could be summed up.